src/server/HtmlDocument.js

```diff
@@ -271,6 +271,20 @@ class HtmlDocument extends React.Component {
                     async={true}
                     src="//www.google-analytics.com/analytics.js"
                 ></script>,
+                <noscript>
+                    <iframe
+                        src="//www.googletagmanager.com/ns.html?id=GTM-5XC34R"
```

Comment:
Can you add it to the environment for staging/demo? I don't have access to that stuff.